Rubberduck.SourceControl/Rubberduck.SourceControl.csproj

```diff
@@ -38,9 +38,9 @@
     <LangVersion>7.1</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="LibGit2Sharp">
-      <HintPath>..\packages\LibGit2Sharp.0.22.0-pre20150516171636\lib\net40\LibGit2Sharp.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="LibGit2Sharp, Version=0.24.0.0, Culture=neutral, PublicKeyToken=7cbde695407f0333, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
```

Comment:
why isn't this private anymore? And why doesn't it require a specific version?

Reply:
This one is interesting. I'm sorry for such a quick change. I actually changed nothing on purpose on this file, VS changed it automatically when I added the new reference to libgit v0.24. But it totally makes sense, according to this stackoverflow: https://stackoverflow.com/questions/16578819/understanding-a-csproj-assembly-reference
If you notice, VS adds all the assembly metadata to the csproj when the reference is added. By default, Specific Version is false, hence, VS adds just a name and a HintPath. If you manually set this value to true then VS adds the whole metadata in the include, and if changing it again to false, VS just keeps the metadata but adds the SpecificVersion directive. What do you think? How should we keep it?
About the private, you got me, VS is quite good but sometimes it betrays you.